india_compliance/patches/post_install/update_sales_rcm_accounts.py

```diff
@@ -0,0 +1,222 @@
+                    "account_name": "sgst_account",
+                    "tax_rate": 9.0,
+                    "account_type": "account_type",
+                },
+                "description": "SGST",
+            },
+            {
+                "account_head": {
+                    "account_name": "cgst_account",
+                    "tax_rate": 9.0,
+                    "account_type": "Tax",
+                },
+                "description": "CGST",
+            },
+            {
+                "account_head": {
+                    "account_name": "Output Tax SGST RCM",
+                    "tax_rate": -9.0,
+                    "account_type": "Tax",
+                },
+                "description": "SGST RCM",
+            },
+            {
+                "account_head": {
+                    "account_name": "Output Tax CGST RCM",
+                    "tax_rate": -9.0,
+                    "account_type": "Tax",
+                },
+                "description": "CGST RCM",
+            },
+        ],
+        "tax_category": "Reverse Charge In-State",
+    },
+    {
+        "title": "Output GST RCM Out-state",
+        "taxes": [
+            {
+                "account_head": {
+                    "account_name": "igst_account",
+                    "tax_rate": 18.0,
+                    "account_type": "Tax",
+                },
+                "description": "IGST",
+            },
+            {
+                "account_head": {
+                    "account_name": "Output Tax IGST RCM",
+                    "tax_rate": -18.0,
+                    "account_type": "Tax",
+                },
+                "description": "IGST RCM",
+            },
+        ],
+        "tax_category": "Reverse Charge Out-State",
+    },
+]
+
+
+def execute():
+    # skipping setup for new installations
+    if frappe.db.get_all(
+        "GST Account", {"account_type": ["=", "Sales Reverse Charge"]}
+    ):
+        return
+
+    companies = frappe.get_all("Company", filters={"country": "India"}, pluck="name")
+    for company in companies:
+        # skipping if account already exists
+        if frappe.get_all(
+            "Account",
+            {
+                "company": company,
+                "account_name": ("in", SALES_RCM_ACCOUNTS),
+            },
+        ):
+            continue
+
+        gst_accounts = get_gst_accounts_by_type(company, "Output")
+        account_names = get_account_names(gst_accounts)
+        tax_categories = get_tax_categories()
+        gst_rate = get_default_gst_rate(gst_accounts)
+
+        for template in TEMPLATE:
+            template["tax_category"] = tax_categories.get(template["tax_category"], "")
+
+            for row in template.get("taxes"):
+                taxes = row["account_head"]
+                update_account_name(gst_accounts, account_names, taxes)
+                update_tax_rate(gst_rate, taxes)
+
+            make_taxes_and_charges_template(
```

Comment:
Issue:
This will create template as per account name as specified in our template. This could be different in GST Settings for Output accounts.

This should be independent patch and we should create taxes and charges using existing output accounts in GST Settings.